src/java/org/apache/cassandra/service/opstate/KeyspaceColumnFamilyKey.java

```diff
@@ -20,29 +20,29 @@
 
 import com.palantir.cassandra.objects.Wrapper;
 
-public class KeyspaceTableKey extends Wrapper<String>
+public class KeyspaceColumnFamilyKey extends Wrapper<String>
```

Comment:
Is there a reason for this rename? - it will make merging Cassandra's trunk more complicated and bloats the pr significantly.

Reply:
I found the word `Table` to be confusing and misleading since Cassandra code refers to them as `ColumnFamily`.

Reply:
This is a painful naming convention but as far as modern Cassandra is concerned Table == Column Family where the latter is the legacy name so we're going backwards with this rename - if you want more details check out [this Blog](https://medium.com/informatics/architecture-and-cassandra-data-model-9eb7a3cda969#85b6) on the internal naming conventions of the storage mode. [CASSANDRA-14340](https://issues.apache.org/jira/browse/CASSANDRA-14340) exists to address the rest of the codebase talking about CFs instead of Tables. As for our fork - It's a bad idea to rename things even if their naming conventions are not the best - it means that if we want to merge a newer branch or try rebasing that process will be trickier.